views/index.pug

```diff
@@ -1,23 +1,19 @@
 html
   head
-    link(rel='stylesheet', href='https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.5/css/bootstrap.min.css', integrity='sha384-AysaV+vQoT3kOAXZkl02PThvDr8HYKPZhNT5h/CXfBThSRXQ6jW5DO2ekP5ViFdi', crossorigin='anonymous')
-    link(rel='stylesheet', href='/box.css')
+    link(rel='stylesheet', href="/css/application.css")
+    link(rel='stylesheet', href="/css/box.css")
```

Comment:
oh! sorry.. im keeping the box file separate for now as I am refactoring the styles with bootstrap. It actually makes sense using their cards, because they have this particular deck div grid which aligns them nicely.